R/multitables.R

```diff
@@ -108,3 +108,27 @@ newMultitable <- function (formula, data, name, ...) {
     u <- crPOST(shojiURL(data, "catalogs", "multitables"), body=toJSON(payload))
     invisible(Multitable(crGET(u)))
 }
+
+#' Import a Multitable
+#'
+#' Allows you to import a multitable from a different dataset
+#' @param data an object of class \code{CrunchDataset} in which to create the
+#' multitable
+#' @param multitable an object of class \code{Multitable} that you want copied
+#' to the new dataset
+#' @param ... Additional multitable attributes to set. Options include
+#' \code{is_public}.
```

Comment:
add "name" as a possible attribute. And if you don't mind, while you're here, swap these \code{} things to either ```` or `[functionName()]` to link to docs, if appropriate.